config-linux.md

```diff
@@ -641,7 +641,9 @@ The values MUST be absolute paths in the [container namespace](glossary.md#conta
 
 ## <a name="configLinuxMountLabel" />Mount Label
 
-**`mountLabel`** (string, OPTIONAL) will set the Selinux context for the mounts in the container.
+**`mountLabel`** (string, OPTIONAL) will set the SELinux context for the mounts in the container.
+If `mountLabel` is specified and SELinux is enabled, the runtime MUST set `mountLabel` for the mounts in the container.
```

Comment:
> I'm not sure how you think about "runtime support SELinux or not"? In my opinion, the support means "When SELinux label is specified, and SELinux is enabled on host, runtime can apply the label correctly", and also we can say the runtime is compliant.

That makes sense to me.  I was thinking of runtime support like [runC's `selinux` build tag][1].  I think we need to require runtimes to respect this property on hosts where SELinux is enabled.  I care less about hosts where SELinux is not available, but would like runtimes to error out during `create` if `mountLabel` (or other SELinux-based option) was set in a config on those hosts.  I think my [proposed wording][2] is fine for this already.  Or do you want to qualify it based on having a [mounted `selinuxfs` in the host mount namespace][3] or some such?

[1]: https://github.com/opencontainers/runc/tree/v1.0.0-rc3#build-tags
[2]: https://github.com/opencontainers/runtime-spec/pull/844#discussion_r118042435
[3]: https://github.com/opencontainers/selinux/blob/v1.0.0-rc1/go-selinux/selinux.go#L118